### PR TITLE
Add a bit of documentation about Webpacker

### PR DESCRIPTION
**Pull request by duncanjbrown**
### Context

Webpacker is sometimes unhelpful when we're working with it, especially for those encountering it for the first time.

### Changes proposed in this pull request

Add some lines to the docs.

### Guidance to review

Do the docs make sense? 

### Link to Trello card

https://trello.com/c/JhgE8jex/699-document-how-to-work-with-webpacker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,23 @@ CHROMEDRIVER_PATH=/path/to/chromedriver bundle exec rake
 
 If you do not have chromedriver installed, first execute
 `bin/install-chromedriver`, which will install it for you and report the path.
+
+## Webpacker
+
+We do not use the Rails asset pipeline: we use
```
Comment (alexdesi) [inline README.md]: `,` instead of `:` ?

Comment (alexdesi) [reply README.md]: Should we specify why we decided to use webpack/webpacker?

File: README.md [hunk 2]
```diff
@@ -28,3 +28,23 @@ CHROMEDRIVER_PATH=/path/to/chromedriver bundle exec rake
...
+[Webpacker](https://github.com/rails/webpacker) for everything. Webpacker
+sometimes doesn't give clear indications of what's wrong when it doesn't work.
+
+**If you have trouble getting assets to compile in dev**, a good place to start
+debugging is by running `bin/webpack`. This will run the webpacker compilation
```
Comment (alexdesi) [inline README.md]: it'd be good to provide one link to the documentation

File: README.md (hunk 1 above)
Comment (duncanjbrown) [reply README.md]: Yes — I think this can be a good ADR https://trello.com/c/oivKkA7C/667-establish-architectural-decision-records-adrs-for-apply

Comment (duncanjbrown) [reply README.md]: > `,` instead of `:` ?

Guilty as charged 🙂. This is a bad colon. Will amend

File: README.md (hunk 2 above)
Comment (duncanjbrown) [reply README.md]: There is a link two lines up — do you think it ought to be made more explicit?

**Comment by duncanjbrown** [comment]
Looked at the file with @alexdesi, added a bit of structure, and this is now ready to merge

**Pull request merged**